utils.groovy

```diff
@@ -199,4 +199,18 @@ def get_push_trigger() {
     ]
 }
 
+def get_artifacts(pipecfg, stream, basearch) {
+    // Get artifacts from config.yaml
+    def artifacts
+    if  (pipecfg.streams."${stream}".artifacts?."${basearch}" != null) {
+        artifacts = pipecfg.streams."${stream}".artifacts."${basearch}"
+        return artifact
```

Comment:
we should remove `return artifact` from this commit. It's wrong (should be `artifacts` and gets later cleaned up anyway).